### DISCUSSION TITLE
Missing files issue in code-server after a clean installation

### DISCUSSION THREAD

**Discussion by LeandroGFC** [coder/code-server#7046] [Q&A]
Hello community,

I am experiencing a recurring problem with my `code-server` installation. It was previously working without issues, but at some point, when accessing it, the screen to enter the password appeared, and after logging in, it only showed a blank screen. Checking the service status, I found several errors indicating that certain essential `.js` files were missing. The errors are the following:

![image](https://github.com/user-attachments/assets/8f266a55-2d3c-458a-ba23-9765cfaed022)

I installed `code-server` using the following commands:

> `VERSION=4.93.1`
> `curl -fOL https://github.com/coder/code-server/releases/download/v$VERSION/code-server_${VERSION}_amd64.deb`
> `sudo apt install ./code-server_${VERSION}_amd64.deb -y`

I have already tried several things without success, including:

- Performing a clean installation of `code-server`.
- Verifying the permissions of the folders.
- Ensuring that all dependencies are correctly installed.

My current setup includes `code-server` installed in `/usr/lib/code-server`. However, it still cannot find these files after installation.

### Actions already taken:

1. Clean installation of `code-server`.
2. Verified permissions (`chown` and `chmod` on the corresponding paths).
3. Checked paths in the configuration file (`~/.config/code-server/config.yaml`).

### Questions:

1. Could it be an issue with incorrect paths in some additional configuration?
2. Is there a specific way to reinstall `code-server` to avoid missing files?
3. Has anyone faced a similar problem and has any recommendations?

Any suggestions or help would be greatly appreciated.

Thanks in advance!

**Comment by code-asher** [comment]
Looks like a case of https://github.com/coder/code-server/issues/2407, which probably needs to be fixed upstream.  It seems that if the main workbench file takes too long to load, it falls back to trying to load development files, which obviously will not exist.
